### PR TITLE
[LiveComponent] "Hooks: Handle Component Behavior" spacing issue in docs

### LINKED ISSUE #2363: [Live] Docs for `PostHydrate`, `PreDehydrate` & `PreReRender`

### PR DESCRIPTION
**Pull request by nehalgajjar**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| Issues        | Fix #2363
| License       | MIT

## [Issue] Fix Line Break / Spacing
image

### CODE REVIEW

**Review by Kocal — APPROVED** [review]

File: src/LiveComponent/doc/index.rst [hunk 1]
```diff
@@ -183,36 +183,23 @@ let's keep going because… things get cooler.
 
 Hooks: Handle Component Behavior
 --------------------------------
-Most of the time, you'll just pass data to your components and 
-let it handle the rest. However, if you need to do something 
-more complex during certain stages of a component's lifecycle, you 
-can take advantage of lifecycle hooks.
+Most of the time, you'll just pass data to your components and let it handle the rest. However, if you need to do something more complex during certain stages of a component's lifecycle, you can take advantage of lifecycle hooks.
```
Comment (smnandre) [inline src/LiveComponent/doc/index.rst]:
```suggestion
Most of the time, you'll just pass data to your components and let it handle the
rest. However, if you need to do something more complex during certain stages of
a component's lifecycle, you can take advantage of lifecycle hooks.
```

Symfony documentation standards has line length requirements: https://symfony.com/doc/current/contributing/documentation/standards.html

Could you update the other changes to stay under 80 chars lines ? 

Thank you!

**Review by smnandre — CHANGES_REQUESTED** [review]
Good catch, but the original issue is probably due to the lack of empty line between the title and the paragraph.

**Comment by smnandre** [comment]
Also, let put this later in the documentation. It's a bit too early right now (all the concepts have not yet been explained that early)

**Review by smnandre — APPROVED** [review]

**Comment by smnandre** [comment]
Thank you @nehalgajjar.

**Pull request merged**